diff --git a/dist/index.js b/dist/index.js
index 1a8c68c0713d41b0239dc8e3895d31437d3bb2e8..f6fec64252c8f1c6225eb37776f6c0d0e2a42c35 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -10474,7 +10474,7 @@ var import_path = require("path");
 // src/constants.ts
 var KIB = 1024;
 var MIB = 1024 * KIB;
-var EDGE_FUNCTION_SIZE_LIMIT = 4 * MIB;
+var EDGE_FUNCTION_SIZE_LIMIT = 10 * MIB;
 var DEFAULT_MAX_UNCOMPRESSED_LAMBDA_SIZE = 250 * MIB;
 var LAMBDA_RESERVED_UNCOMPRESSED_SIZE = 25 * MIB;
 
